### DISCUSSION TITLE
-geometry offset is not working as expected

### DISCUSSION THREAD

**Discussion by bullmoose20** [ImageMagick/ImageMagick#5681] [Help] [answered]
magick back.jpg logo.jpg -gravity center -geometry +200+200 -background transparent -extent 2000x3000 -layers Flatten result.jpg

![image](https://user-images.githubusercontent.com/12549033/196833969-b38e6da5-f278-475f-9324-fc8426cf8121.jpeg)

It centres but then won't apply the geometry to offset the logo...

**Comment by snibgo** [comment]
"`-layers XX`" is not sensitive to "`-geometry`". Instead, use "`-page`" or similar with "`-layers XX`", or "`-geometry`" with "`-composite`".

**Reply by bullmoose20** [reply]
![Spotify](https://user-images.githubusercontent.com/12549033/196930325-ecdd32f5-c0b9-4b96-b589-4c421c212bd6.jpg)

not sure what i am doing wrong. see below for the two files.

**Reply by bullmoose20** [reply]
`magick composite -gravity center -geometry +0-400 logo.png back.jpg result.jpg` 
this worked

**Comment by bullmoose20** [comment]
![back](https://user-images.githubusercontent.com/12549033/196929541-447848fb-de93-4b79-bb18-f7b7bd945e3e.jpg)
![logo](https://user-images.githubusercontent.com/12549033/196929545-c309424d-33e7-42cf-b03d-ce7269cb6d8b.png)

hmmm so i have tried what you are suggesting, hiwever i am clearly unable to succeed in moving the logo up from the gravity center by 200px

**Comment by snibgo** [comment, marked as answer]
>  i am clearly unable to succeed in moving the logo up from the gravity center by 200px

Please show your input files, and your command(s).

**Reply by bullmoose20** [reply]
`magick composite -gravity center -geometry +0-400 logo.png back.jpg result.jpg`
this worked
